### DISCUSSION TITLE
Error: Cannot find module 'C:\Users\kasani\.yarn\releases\yarn-3.0.2.cjs'

### DISCUSSION THREAD

**Discussion by ankasani** [yarnpkg/berry#3653] [Q&A]
Keep getting below error continuously on my windows machine.

this path C:\Users\kasani\.yarn > itself is not there, not sure where it got it from.t from.

Uninstall and reinstall the yarn, still the same issue

internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module 'C:\Users\kasani\.yarn\releases\yarn-3.0.2.cjs'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Please can any one help on this ?

**Comment by merceyz** [comment]
Your `yarnPath` is pointing to a path that doesn't exist, check `~/.yarnrc.yml` and remove the `yarnPath` config
